.github/workflows/check_make_vtadmin_authz_testgen.yml

```diff
@@ -6,10 +6,22 @@ jobs:
     name: Check Make vtadmin_authz_testgen
     runs-on: ubuntu-latest
     steps:
+    - name: Check if workflow needs to be skipped
+      id: skip-workflow
+      run: |
+        skip=0
+        if [[ "${{github.event.pull_request}}" ==  "" ]] && [[ "${{github.ref}}" != "refs/heads/main" ]] && [[ ! "${{github.ref}}" =~ "refs/heads/release-.*\.0" ]] && [[ ! "${{github.ref}}" =~ "refs/tags/.*" ]]; then
+          skip=1
```

Comment:
Another nit, but I think we should be consistent and use `true` / `false` like we already are in the workflows rather than 1/0. It's also more explicit for humans reading the code, which I like.

Reply:
Done too! :)